greg-maxwell/2015-11-09-gmaxwell-mining-and-block-size-etc.md

```diff
@@ -8,8 +8,6 @@ tags: ['mining']
 
 # Mining, block size, etc.
 
-<https://www.youtube.com/watch?v=RguZ0_nmSPw>
```

Comment:
Getting rid of these links would be a mistake I think - we can preface with wayback machine and maybe in a future pass we create our own archive. But if we lose the link, they will be lost forever.

Reply:
Thank you for the review. Below is the summary of actions to take:

- rename the `video` tag to `media`
- add remaining audio links
- revert removed dead links
- cleanup the commit history

Please let me know if I did not get any. Thank you

Reply:
I first mentioned dead links to Peter, but I was referring only to video dead links. It's true that some of the non-video links might be accessible through the wayback machine, but I don't believe that the same is true for videos. 

Regarding the dead video links, I was mostly concern of exchanging dead links with ones that work. If there is no working link, keeping them as historical data makes sense.